tox.ini

```diff
@@ -74,6 +77,7 @@ extras =
     optimized
 passenv =
     PYPY_GC_MAX
+    PYPY_GC_MIN
```

Comment:
Since the `optimized` workflow uses CPython, I am not sure if we need to pass `PYPY_GC_MAX` and `PYPY_GC_MIN`